libr/core/canal.c

```diff
@@ -835,6 +835,8 @@ static void print_hint_h_format(RAnalHint* hint) {
 	HINTCMD (hint, syntax, " syntax='%s'", false);
 	HINTCMD (hint, immbase, " immbase=%d", false);
 	HINTCMD (hint, esil, " esil='%s'", false);
+	if(hint->jump != UT64_MAX)
```

Comment:
Missing space before ( and add braces around the printf, and add another space before the ( in there. This comment applies to the other lines too